src/ConfigPostProcessor.php

```diff
@@ -251,9 +259,13 @@ private function replaceDependencyConfiguration(array $config)
 
         $config = $this->replaceDependencyInvokables($config);
         $config = $this->replaceDependencyFactories($config);
-        $delegators = isset($config['delegators']) ? $this->replaceDependencyDelegators($config['delegators']) : [];
-        if ($delegators) {
-            $config['delegators'] = $delegators;
+
+        foreach ($config as $key => $data) {
+            if (in_array($key, ['aliases', 'invokables', 'factories'], true)) {
```

Comment:
Array of keys is now used in two places, so we might want to extract it somewhere or improve this logic a bit.

The key observation here is that we want to apply normal replacement rules on all other keys than `aliases`, `invokables`, `factories`.

Reply:
Moved em into a constant.
Can change to a private static if we dont want to spread this to public.